adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+
+LETTER_POOL = {"A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, 
+    "H": 2, "I": 9, "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8,
+    "P": 2, "Q": 1, "R": 6, "S": 4, "T": 6, "U": 4, "V": 2, "W": 2, "X": 1,
+    "Y": 2, "Z": 1}
+LETTERS = ["A","B","C","D","E","F","G","H","I","J","K","L", "M", "N","O","P","Q","R","S","T","U","V","W","X","Y","Z"]
+
+SCORE_CHART = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J", "X"],
+    10: ["Q", "Z"]
+}
 def draw_letters():
-    pass
+    # make copy of the letter bank and letter pool constants
+    letters = LETTERS.copy()
+    letter_pool = LETTER_POOL.copy()
+    # initiate hand empty list
+    hand = []
+    while len(hand) < 10:
+        letter = random.choice(letters)
+        for key, value in letter_pool.items():
+            if letter == key and value > 0:
+                letter_pool[key] -= 1
+                hand.append(letter)
+    return hand
+
 
 def uses_available_letters(word, letter_bank):
```

Comment:
This approach works but is kind of lengthy you could use `deepcopy` like you did in the previous function and do something like 
```python
def uses_available_letters(word, letter_bank):
    letter_bank_copy = copy.deepcopy(letter_bank)

    for letter in word:
        if letter in letter_bank_copy:
            letter_bank_copy.remove(letter)
        else:
            return False
    
    return True
```